### PR TITLE
New attribute proposed for WasteContainer: weight

### PR DESCRIPTION
**Pull request by cayetanobv**
Hi team,

With this PR we are proposing a new attribute (weight) for WasteContaniner entity.

Thanks!

Cayetano Benavent, Geographica

### CODE REVIEW

File: WasteManagement/WasteContainer/doc/spec.md [hunk 1]
```diff
@@ -33,6 +33,19 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Allowed values: Interval [0,1]. 
     + Optional
     
++ `weight` : Weight of the container load.
+    + Attribute type: [Number](http://schema.org/Number)
```
Comment (jmcanterafonseca) [inline WasteManagement/WasteContainer/doc/spec.md]: are you sure we are going to be able to know the weight of the container load or just the load of all the set i.e. container + its load ?

Comment (jmcanterafonseca) [inline WasteManagement/WasteContainer/doc/spec.md]: as per latest discussions I propose the following

name the attribute as `cargoWeight` , to denote precisely that we are talking about the weight of the container's cargo. 

We need as well to link the TimeInstant metadata to its description, please check other attributes. 

BTW we are about to renaming `dateUpdated` to `timestamp` to align with GSMA's specs but that will be done in another PR made by us.

**Comment by cayetanobv** [comment]
Hi @jmcanterafonseca , I commited requested changes. Thanks!

File: WasteManagement/WasteContainer/doc/spec.md [hunk 2]
```diff
@@ -34,6 +34,19 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Allowed values: Interval [0,1]. 
     + Optional
     
++ `cargoWeight` : Weight of the container load.
+    + Attribute type: [Number](http://schema.org/Number)
+    + Attribute metadata:
+        + `dateUpdated`: Timestamp when the last update of the attribute happened.
+            + Type: [DateTime](http://schema.org/DateTime)
+        + `TimeInstant` : Timestamp saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
```
Comment (jmcanterafonseca) [inline WasteManagement/WasteContainer/doc/spec.md]: you forgot a link to timeinstant definition, check other attributes, thanks!

Comment (cayetanobv) [inline WasteManagement/WasteContainer/doc/spec.md]: Upsss sorry... I already commited this change. Thanks @jmcanterafonseca

**Comment by jmcanterafonseca** [comment]
thanks, merged

**Pull request merged**